cmd/gitops/add/app/cmd.go

```diff
@@ -118,6 +118,15 @@ func runCmd(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to create app service: %w", err)
 	}
 
+	wegoConfig, err := kubeClient.GetWegoConfig(ctx, params.Namespace)
+	if err != nil {
+		return fmt.Errorf("failed getting wego config")
+	}
+
+	if params.ConfigRepo == "" {
+		params.ConfigRepo = wegoConfig.ConfigRepo
```

Comment:
I was under the impression that we were getting rid of params.ConfigRepo in favor of only pulling it from the wego config. Do we have coverage for when the params and the wegoConfig do not match?

Reply:
yeah, I think we should just remove that params like in the other methods, it doesn't not make sense